include/rocksdb/env.h

```diff
@@ -769,6 +777,12 @@ class WritableFile {
   // PositionedAppend, so the users cannot mix the two.
   virtual Status Append(const Slice& data) = 0;
 
+  // Append data with verification information
+  virtual Status AppendWithVerify(
+      const Slice& data, const DataVerificationInfo& /* verification_info */) {
+    return Append(data);
+  }
```

Comment:
Do we need to add it in the legacy WritableFile, or adding to FSWritableFile() would be sufficient?